roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -17,6 +17,25 @@
 - name: Gather service facts
   ansible.builtin.service_facts:
 
+- name: Check if firewalld service exists
+  ansible.builtin.command: systemctl status firewalld
```

Comment:
Direct usages of `ansible.builtin.command` are discouraged if a module/plugin is available for the same purpose.
In this particular case, the information you want to fetch, if `firewalld` is present or not, is already available thanks to https://github.com/openstack-k8s-operators/edpm-ansible/pull/690/files#diff-2c5bbb52f169a24c08c751de07b1e24779125b4e3712c542f5b0cf50f298ab6aR18
Leverage those facts for this purpose like this:

```yaml
    - name: Ensure firewalld service is disabled
      when:
        - ansible_facts.services["firewalld.service"] is defined
      ansible.builtin.systemd: <....>
        
```

Reply:
done